openapi3/schema_validation_settings.go

```diff
@@ -28,10 +30,19 @@ func MultiErrors() SchemaValidationOption {
 func VisitAsRequest() SchemaValidationOption {
 	return func(s *schemaValidationSettings) { s.asreq, s.asrep = true, false }
 }
+
 func VisitAsResponse() SchemaValidationOption {
 	return func(s *schemaValidationSettings) { s.asreq, s.asrep = false, true }
 }
 
+func DisableFormatValidation() SchemaValidationOption {
```

Comment:
You should flip the default (and mention this in README) so maybe name it `EnableFormatValidation`.

Please add some documentation, such as "EnableFormatValidation setting makes Validate not return an error when validating documents that mention schema formats that are not defined by the OpenAPIv3 specification".

Reply:
Flipped and documentation added